src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
The class 'ColorSupplier' should have an access modifier. Preferably, it should be public if you want to use it outside its package.